### PR TITLE
Update and rename cloud_security_investigator.md to cloud_siem_invest…

### PR DESCRIPTION
**Pull request by Partha-Naidu**
…igator.md

### What does this PR do?

- Renames Cloud Security Investigator to Cloud SIEM Investigator
- Adds alias
- Updates nav link

Preview link: https://docs-staging.datadoghq.com/may/investigator-preview/security_platform/cloud_siem/cloud_siem_investigator

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/security_platform/cloud_siem/cloud_siem_investigator.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+- link: "/cloud_siem/explorer/"
+  tag: "Documentation"
+  text: "Learn about the Security Signals Explorer"
+---
+
+{{< beta-callout url="#" btn_hidden="true">}}
+  The Cloud SIEM Investigator is in beta.
+{{< /beta-callout >}} 
+
+<div class="alert alert-warning">Cloud SIEM Investigator currently only supports AWS CloudTrail logs.</div>
```
Comment (alai97) [inline content/en/security_platform/cloud_siem/cloud_siem_investigator.md:16]: I would move this this banner below the Overview header.

File: content/en/security_platform/cloud_siem/cloud_siem_investigator.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+## Overview
+
+When a security signal alerts on suspicious activity by a user or a resource, common questions asked as part of the investigation could include:
```
Comment (alai97) [inline content/en/security_platform/cloud_siem/cloud_siem_investigator.md:20]:
```suggestion
When a security signal alerts on suspicious activity by a user or a resource, some commonly asked questions during the investigation include:
```

File: content/en/security_platform/cloud_siem/cloud_siem_investigator.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+- What users have interacted with this resource?
+
+For example, suppose you receive a security signal that someone changed the configuration of an AWS S3 bucket so that it is accessible by everyone, but the action was taken by an assumed role. To investigate, look into who took the action and what other activities they did recently, as that could indicate compromised credentials.
+
+The Cloud SIEM Investigator provides a graphical interface for you to pivot from one affected entity to another, so that you can see user behavior and its impact on your environment.
+
+
+## Visualize and investigate the activity
+
+1. Navigate to the [Cloud SIEM Investigator][1]. Alternatively, you can get to it from **Security** > **Cloud SIEM**, and then click the **Investigator** tab.
```
Comment (alai97) [inline content/en/security_platform/cloud_siem/cloud_siem_investigator.md:34]:
```suggestion
1. Navigate to **Security** > **Cloud SIEM** and click the [**Cloud Security Investigator** tab][1]. 
```

File: content/en/security_platform/cloud_siem/cloud_siem_investigator.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+The Cloud SIEM Investigator provides a graphical interface for you to pivot from one affected entity to another, so that you can see user behavior and its impact on your environment.
+
+
+## Visualize and investigate the activity
+
+1. Navigate to the [Cloud SIEM Investigator][1]. Alternatively, you can get to it from **Security** > **Cloud SIEM**, and then click the **Investigator** tab. 
+
+2. Select an entity type in the **In** field dropdown.
+
+3. Select a entity or enter a specific entity name in the **Investigate** field to see a graph of the activities associated with the entity. For the Assumed Role entity, select an `AccessKeyID` or enter an `AccessKeyID` in the **for** field.
```
Comment (alai97) [inline content/en/security_platform/cloud_siem/cloud_siem_investigator.md:38]:
```suggestion
3. Select an entity or enter a specific entity name in the **Investigate** field to see a graph of the activities associated with the entity. For the **Assumed Role** entity, select an `AccessKeyID` or enter an `AccessKeyID` in the **for** field. 
```

File: content/en/security_platform/cloud_siem/cloud_siem_investigator.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+## Visualize and investigate the activity
+
+1. Navigate to the [Cloud SIEM Investigator][1]. Alternatively, you can get to it from **Security** > **Cloud SIEM**, and then click the **Investigator** tab. 
+
+2. Select an entity type in the **In** field dropdown.
+
+3. Select a entity or enter a specific entity name in the **Investigate** field to see a graph of the activities associated with the entity. For the Assumed Role entity, select an `AccessKeyID` or enter an `AccessKeyID` in the **for** field. 
+
+4. Click on a node and select **Show list of logs** or **View in Log Explorer** to see the related logs. If you click on a service node, click **Investigate service** to pivot to the Investigator view for that service. Use the **and filter by** dropdown to filter by actions.
```
Comment (alai97) [inline content/en/security_platform/cloud_siem/cloud_siem_investigator.md:40]:
```suggestion
4. Click on a node and select **Show list of logs** or **View in Log Explorer** to see the related logs. If you click on a service node, click **Investigate service** to pivot to the Investigator view for that service. Use the **and filter by** dropdown menu to filter by actions.
```

File: content/en/security_platform/cloud_siem/cloud_siem_investigator.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+You can also go to the Cloud SIEM Investigator directly from a security signal. In the security signal panel, click **Investigate user activity**, where `user` is the user identity in question, to see the Investigator view filtered to the specific user identity.
```
Comment (alai97) [inline content/en/security_platform/cloud_siem/cloud_siem_investigator.md:42]:
```suggestion
You can also navigate to the Cloud SIEM Investigator directly from a security signal. In the security signal panel, click **Investigate user activity** (where `user` is the user identity in question) to see the Investigator view filtered to the specific user identity.
```

File: content/en/security_platform/cloud_siem/cloud_siem_investigator.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+- What other actions did the user take around that specific time frame? 
+- What are all the actions taken on a resource by the user?
+- What users have interacted with this resource?
+
+For example, suppose you receive a security signal that someone changed the configuration of an AWS S3 bucket so that it is accessible by everyone, but the action was taken by an assumed role. To investigate, look into who took the action and what other activities they did recently, as that could indicate compromised credentials.
+
+The Cloud SIEM Investigator provides a graphical interface for you to pivot from one affected entity to another, so that you can see user behavior and its impact on your environment.
+
+
+## Visualize and investigate the activity
```
Comment (alai97) [inline content/en/security_platform/cloud_siem/cloud_siem_investigator.md:32]:
```suggestion
## Visualize and investigate user activity
```

**Review by alai97 — CHANGES_REQUESTED** [review]
Some copy suggestions!

File: content/en/security_platform/cloud_siem/cloud_siem_investigator.md (hunk 7 above)
Comment (maycmlee) [reply content/en/security_platform/cloud_siem/cloud_siem_investigator.md:32]: Will this as is because it could be user or service activity.

**Review by maycmlee — APPROVED** [review]

**Pull request merged**